### PR TITLE
Spidermon schema

### PR DESCRIPTION
**Pull request by andersonberg**
This PR will handle https://github.com/okfn-brasil/querido-diario/issues/267 also adding and enabling Spidermon.

### CODE REVIEW

**Comment by andersonberg** [comment]
While working on this PR, I found a bug in the Cuiaba Spider that was returning `None` for the edition date. Since it's a small change I find it better to include in this PR

File: processing/data_collection/gazette/schema.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "date": {
+            "description": "Date of the gazzete",
+            "type": "string"
```
Comment (rennerocha) [inline processing/data_collection/gazette/schema.json:7]: **issue**: Gazette date is a well defined string with YYYY-MM-DD. It would be a good idea to match this pattern when checking this field.

File: processing/data_collection/gazette/settings.py [hunk 2]
```diff
@@ -10,3 +10,13 @@
 }
 FILES_STORE = "/mnt/data/"
 QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
+
+EXTENSIONS = {
+    "spidermon.contrib.scrapy.extensions.Spidermon": 500,
+}
+SPIDERMON_ENABLED = True
+ITEM_PIPELINES = {
```
Comment (rennerocha) [inline processing/data_collection/gazette/settings.py:18]: **issue (blocking)**: We already have a list of ITEM_PIPELINES defined. This change will disable all other pipelines.

File: processing/data_collection/gazette/spiders/mt_cuiaba.py [hunk 3]
```diff
@@ -42,7 +42,7 @@ def parse(self, response):
             edition_date = edition["publication_date"]
 
             yield Gazette(
-                date=parse(edition_date, languages=["pt"]),
+                date=parse(edition_date),
```
Comment (rennerocha) [inline processing/data_collection/gazette/spiders/mt_cuiaba.py:45]: **suggestion**: This change is not related to Spidermon, so it shouldn't be here. Also when parsing a date using dateparser, it is good practice to pass the language to make the parsing more reliable.

**Review by rennerocha — CHANGES_REQUESTED** [review]
@andersonberg thanks for the PR! I added a few comments but we have few changes before getting this merged.

File: processing/data_collection/gazette/schema.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "is_extra_edition": {
+            "description": "Determines if the edition of the gazzete is extra or not",
+            "type": "boolean"
+        },
+        "territory_id": {
+            "type": "string"
+        },
+        "power": {
+            "type": "string",
+            "examples": ["executive", "legislative"]
```
[the remark this answers is not in the document]
Comment (andersonberg) [reply processing/data_collection/gazette/schema.json:24]: this is only a list with examples, but I can add a `enum` with possible values

File: processing/data_collection/gazette/schema.json (hunk 1 above)
Comment (andersonberg) [reply processing/data_collection/gazette/schema.json:7]: that's a good idea, I thought the date were a "datetime" object

File: processing/data_collection/gazette/spiders/mt_cuiaba.py (hunk 3 above)
Comment (andersonberg) [reply processing/data_collection/gazette/spiders/mt_cuiaba.py:45]: ok I'll add a new PR to fix this. Maybe change the language, because using "pt", dateparser were returning `None`

**Review by rennerocha — APPROVED** [review]
@andersonberg 
I did a few small changes to make Spidermon work correctly.

**Comment by andersonberg** [comment]
Thank you @rennerocha! Looks good to me
